shorebird.md

```diff
@@ -0,0 +1,48 @@
+# Shorebird's change in the Flutter Repo 🐦
```

Comment:
Would be nice to move this to either our docs site or `_shorebird` in the [docs directory](https://github.com/shorebirdtech/_shorebird/tree/main/docs) in order to keep our docs co-located. I worry that we'll lose track of all the places where we have docs otherwise 🤷

Reply:
Agreed! I added a PR there and will close this one.